.changeset/ninety-meals-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/coinmetrics-adapter': patch
+---
+
+Add ETH to valid quotes for WS endpoint
```

Comment:
Looks like we're also adding BTC to the array of valid quotes, was that intended?

Reply:
It was, updated the changelog 👍